docs/UserGuide.md

```diff
@@ -263,26 +323,54 @@ Format: `i-restock`
 
 #### 3.1 Adding an employee : `c-add`
 
-Adds an employee to the contact list.
+Adds an employee to the Employee Directory.
 
-Format: `c-add n/NAME p/PHONE_NUMBER e/EMERGENCY_CONTACT a/ADDRESS [t/TAG] ...`
+Format: `c-add n/NAME p/PHONE_NUMBER e/EMERGENCY_CONTACT a/ADDRESS [t/TAG]`
 
-<div markdown="span" class="alert alert-primary">:bulb: 
 
-**Tip:**
-An employee can have any number of tags (including 0)
-</div>
+<div markdown="block" class="alert alert-info">
 
-Examples:
-* `c-add n/John Doe p/98765432 e/81234567 a/Blk 123 ABC Road`.
-* `c-add n/Betsy Crowe e/81234567 p/89007413 a/Blk 120 ABC Road t/Friday t/PartTime`.
+**:information_source: Note:** <br>
+In tCheck, you may add an employee to the Employee Directory. When you add an employee, you need to provide
+his/her name, phone number, emergency contact, and address.<br>
+Tag is optional. <br>
+PHONE_NUMBER and EMERGENCY_CONTACT must be 8-digit number and start with 8 or 9.<br>
+An employee can have any number of tags (including 0).<br>
```

Comment:
This should not be placed in a box, as they explain the parameter restriction. It should be in bullet points under the format line